tests/test_timefuncs.py

```diff
@@ -46,6 +46,17 @@ def test_types_comparison() -> None:
     tsr2: bool = ts > ts2
 
 
+def test_tupes_timestamp_series_comparisons()->None:
```

Comment:
Can you add a line here that says #GH 27 to point to the original issue?

Reply:
Done!